### ISSUE TITLE
_data/irc_versions.yml should refer to drafts as drafts

### ISSUE THREAD

**Issue by Zarthus** [ircv3/ircv3.github.io#131]
(if you ask me, at least!)

Make the following YML:

```yaml
v3.3:
  name: IRCv3.3 Draft
  note: >
    **These specifications are drafts and are NOT completed IRCv3 specs.**

    These caps are generally implemented with the `draft/` prefix until they are made into released IRCv3 specifications.
  specs:
    message-tags:
```

in either:
```yaml
draft/v3.3:  # <--
  name: IRCv3.3 Draft
  note: >
    **These specifications are drafts and are NOT completed IRCv3 specs.**

    These caps are generally implemented with the `draft/` prefix until they are made into released IRCv3 specifications.
  specs:
    message-tags:
```

or
```yaml
v3.3:
  name: IRCv3.3 Draft
  note: >
    **These specifications are drafts and are NOT completed IRCv3 specs.**

    These caps are generally implemented with the `draft/` prefix until they are made into released IRCv3 specifications.
  specs:
    draft/message-tags:  # <--
```

This change stops accidentally advertising a draft as a finished product.

**Comment by jwheare** [comment]
Given we're moving away from the `3.3` naming scheme, this section will need renaming anyway. Can changed it to just `drafts`.

**Comment by Zarthus** [comment]
And keep an infinitely long list of all drafts that ever existed? At what point do you remove old drafts?

**Comment by jwheare** [comment]
They'll get removed when they're published or rejected.

**Comment by DanielOaks** [comment]
Makes sense, I'll do this along with the support page overhauls

**Comment by DanielOaks** [comment]
Implemented by 3441db525f12470ee94bbf6ec60fdd4efabd9f2a, which will be merged into master at some point. At the same time, that also makes them show as `draft/x` on the support pages

**Comment by jwheare** [comment]
Closing, handled in #135
